lib/generators/react_on_rails/templates/node/base/client/node/server.js

```diff
@@ -2,7 +2,7 @@ const net = require('net');
 const fs = require('fs');
 
 const bundlePath = '../../app/assets/webpack/';
-let bundleFileName = 'webpack-bundle.js';
+let bundleFileName = 'server-bundle.js';
```

Comment:
@AlexKVal This is supposed to work with the default generator. I don't think you should change the file name.

CC: @jbhatab